packages/react-components/src/components/utils/useLongPress.ts

```diff
@@ -6,21 +6,20 @@ import { useMemo, useRef, useState, useCallback, useEffect } from 'react';
 /**
  * @private
  */
-export default function useLongPress(
-  onClick: () => void,
-  onLongPress: () => void,
-  touchEventsOnly: boolean
-): {
-  handlers: {
-    onClick: () => void;
-    onMouseDown: () => void;
-    onMouseUp: () => void;
-    onTouchStart: () => void;
-    onTouchEnd: () => void;
-    onKeyDown: () => void;
-    onKeyUp: () => void;
-  };
+export default function useLongPress(props: {
+  onClick?: () => void;
```

Comment:
nit: move optional props to the bottom.
onLongPress
onClick?
touchEventsOnly?